tfe/data_source_organization_tags.go

```diff
@@ -0,0 +1,81 @@
+package tfe
+
+import (
+	"fmt"
+	"strconv"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFEOrganizationTags() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFEOrganizationTagsRead,
+
+		Schema: map[string]*schema.Schema{
+			"organization": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"workspace_count": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceTFEOrganizationTagsRead(d *schema.ResourceData, meta interface{}) error {
+	tfeClient := meta.(ConfiguredClient)
+
+	organizationName, err := tfeClient.schemaOrDefaultOrganization(d)
+	if err != nil {
+		return err
+	}
+
+	var tags []map[string]string
+
+	options := tfe.OrganizationTagsListOptions{}
+	for {
+		organizationTagsList, err := tfeClient.Client.OrganizationTags.List(ctx, organizationName, &options)
+		if err != nil {
+			return fmt.Errorf("Error retrieving organization tags: %w", err)
+		}
+
+		for _, orgTag := range organizationTagsList.Items {
+			tag := map[string]string{"id": orgTag.ID, "name": orgTag.Name, "workspace_count": strconv.Itoa(orgTag.InstanceCount)}
```

Comment:
I'm guessing this was why you opted to make the instance count a TypeString instead of a TypeInt, right? But it's fine to build the tag as a `map[string]interface{}` and use different value types for different keys — the schema helper package is designed to help recover from that kind of type erasure. 

For an example of an existing resource that does exactly this, take a look at the `resourceTFETeamAccessRead()` function: `workspace_locking` and `run_tasks` are `TypeBool` in the schema, and the rest are `TypeString`.

https://github.com/hashicorp/terraform-provider-tfe/blob/f805460f41b8a77266b66eb03fe34d6be61c595d/tfe/resource_tfe_team_access.go#L238-L246

(As a bonus, note that `dataSourceTFETeamAccessRead()` re-uses this same function to build a list of object-typed values, which is about what you're doing.)